test/e2e/framework/metrics_util.go

```diff
@@ -53,7 +53,7 @@ const (
 
 	// We set a higher threshold for list apicalls as they can take more time when
 	// the list is really big. For eg. list nodes in a 5000-node cluster.
-	apiListCallLatencyThreshold time.Duration = 2 * time.Second
+	apiListCallLatencyThreshold time.Duration = 5 * time.Second
```

Comment:
I'm a bit against this. As I said offline, can we use 5s when the cluster is actually large? [as an approximation of the new SLO initially].

I mean, let's use 1s if the clusters is <= 500nodes and 5s otherwise for example. Or sth like that.